### ISSUE TITLE
Face mask issue

### ISSUE THREAD

**Issue by windspirit95** [omasaht/headpose-fsanet-pytorch#6]
Hi,
I really appreciate your open source as well as pre-trained models. They work really good.
However, it only worked on unmaked-face. As you know the current worldwide situation, everyone wears the mask that makes the prediction on yaw, pitch, roll is incorrect.
Is there any dataset or pretrained model to solve this issue? Thank you.

**Comment by ohtlab** [comment]
Hello,
I am not aware of face mask datasets so I can't help you with that, sorry. You can try asking the original author in their repository: https://github.com/shamangary/FSA-Net
